### PR TITLE
docs: remove demo app

### PR DESCRIPTION
**Pull request by arnaudforaison**
## Related issue

Into contributing file, there're always a reference to example/demo app. So this PR remove it.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -28,24 +27,14 @@ This repository contains some projects used to display, explain components built
...
+npm ci
 ```
 
 ```sh
 cd storybook/storybook
-npm i
-```
-
-```sh
-cd storybook/styles
```
Comment (gcruchon) [inline CONTRIBUTING.md:47]: Why not remove line 24 as well?

**Review by gcruchon — COMMENTED** [review]
Nice PR! I've just read it for the first time and did not understand why demo was not here :-) 

Also, would be good to explain why Node v12 is not yet supported too, what do you think?
Also, the commit name could include the "storybook/styles" removal as well, I guess.

File: CONTRIBUTING.md (hunk 1 above)
Comment (samuel-gomez-axa) [reply CONTRIBUTING.md:47]: already done here : https://github.com/AxaGuilDEv/react-toolkit/pull/692

**Comment by arnaudforaison** [comment]
Close by #692 & Commit  228ff3e93874591efbbe7643bb16660147dd315c

**Comment by arnaudforaison** [comment]
Close by #692 & Commit 228ff3e93874591efbbe7643bb16660147dd315c

**Pull request closed without merging**